firefox-ios/Client/Frontend/Library/Bookmarks/Edit Bookmark/EditBookmarkViewModel.swift

```diff
@@ -96,9 +104,10 @@ class EditBookmarkViewModel {
         node = node?.copy(with: bookmarkTitle, url: url)
     }
 
-    func saveBookmark() {
-        guard let selectedFolder, let node else { return }
-        Task { @MainActor [weak self] in
+    @discardableResult
+    func saveBookmark() -> Task<Void, Never>? {
```

Comment:
Added the `Task` as discardable result to be able to test this async code without completion handler